aggregation/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM public.ecr.aws/lambda/python:latest
+
+COPY requirements.txt .
+RUN pip3 install -r requirements.txt --target "${LAMBDA_TASK_ROOT}"
+
+COPY .env .
+COPY lambda.py ${LAMBDA_TASK_ROOT}
+COPY PGConnection.py .
+
+ENV PYTHONUNBUFFERED True
```

Comment:
what does this ENV variable mean? just interested

Reply:
Makes it so the logs a dumped straight from the image, rather than going into a buffer first, meaning that you get the log messages as they occur